src/coreComponents/constitutive/relativePermeability/BrooksCoreyBakerRelativePermeability.hpp

```diff
@@ -40,12 +40,14 @@ class BrooksCoreyBakerRelativePermeabilityUpdate final : public RelativePermeabi
                                               arrayView1d< integer const > const & phaseTypes,
                                               arrayView1d< integer const > const & phaseOrder,
                                               arrayView3d< real64, relperm::USD_RELPERM > const & phaseRelPerm,
-                                              arrayView4d< real64, relperm::USD_RELPERM_DS > const & dPhaseRelPerm_dPhaseVolFrac )
+                                              arrayView4d< real64, relperm::USD_RELPERM_DS > const & dPhaseRelPerm_dPhaseVolFrac,
+                                              arrayView2d< real64, compflow::USD_PHASE > const & phaseTrapped )
     : RelativePermeabilityBaseUpdate( phaseTypes,
                                       phaseOrder,
+                                      phaseMinVolumeFraction,
```

Comment:
```suggestion
                                      phaseMinVolFrac,
```
for consistency with the other variable names in the call (`dPhaseRelPerm_dPhaseVolFrac`, `phaseTrappedVolFrac`).